examples/pytorch/dino.py

```diff
@@ -11,12 +11,12 @@
 from lightly.models.utils import update_momentum
 
 class DINO(torch.nn.Module):
-    def __init__(self, backbone, input_dim):
+    def __init__(self, backbone, input_dim, freeze_last_layer, norm_last_layer):
         super().__init__()
         self.student_backbone = backbone
-        self.student_head = DINOProjectionHead(input_dim, 512, 64, 2048)
+        self.student_head = DINOProjectionHead(input_dim, 512, 64, 2048, freeze_last_layer, norm_last_layer)
         self.teacher_backbone = copy.deepcopy(backbone)
-        self.teacher_head = DINOProjectionHead(input_dim, 512, 64, 2048)
+        self.teacher_head = DINOProjectionHead(input_dim, 512, 64, 2048, freeze_last_layer, norm_last_layer)
```

Comment:
Here we don't have to pass the `freeze_last_layer` because the teacher head is not frozen.

Reply:
Noted thanks !